src/test/groovy/pl/allegro/tech/build/axion/release/domain/VersionResolverTest.groovy

```diff
@@ -105,4 +105,84 @@ class VersionResolverTest extends RepositoryBasedTest {
         version.version.toString() == '2.0.0'
         version.snapshot
     }
+		
+		def "should return the highest version from the tagged versions"() {
```

Comment:
tagged version is already on the latest commit, i don't see what is the purpose of this test - if you moved highest version to some tag back in the history it would make much more sense to me